src/pages/Defi/views/StakingVaults.tsx

```diff
@@ -12,9 +16,48 @@ const DefiHeader = () => {
   )
 }
 
+const FoxFarmCTA = () => {
+  const ethLogoSrc = 'https://assets.coincap.io/assets/icons/256/eth.png'
+  const foxLogoSrc = 'https://assets.coincap.io/assets/icons/256/fox.png'
+  return (
+    <Card variant='outline' my={1}>
+      <Stack
+        direction={{ base: 'column', xl: 'row' }}
+        alignItems='center'
+        justifyContent='space-between'
+        px={10}
+        py={4}
+      >
+        <Flex alignItems='center'>
+          <RawText alignSelf='end' fontSize={30} mr={-2} mb={{ base: '2', xl: '-2' }} zIndex={1}>
```

Comment:
As the `+` letter makes it being on 3 lines, I would be proactive and reduce the font-size of the tractor like this on mobile: 

```suggestion
            <RawText lineHeight={'1.1'} fontSize={{ base: 25, sm: 30 }} zIndex={1}>
```

So we are able to have a 2 line text!

Reply:
![image](https://user-images.githubusercontent.com/14963751/167686107-85d89aa2-2d8e-4dd7-8e79-e605b5ce5f2c.png)